tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -1807,15 +1807,8 @@ TEST_F(DataserviceReadVersionedLayerClientTest,
 
   ASSERT_NE(future.wait_for(kWaitTimeout), std::future_status::timeout);
   PrefetchTilesResponse response = future.get();
-  ASSERT_TRUE(response.IsSuccessful()) << response.GetError().GetMessage();
-  ASSERT_FALSE(response.GetResult().empty());
-  for (auto tile_result : response.GetResult()) {
-    std::string str = tile_result->tile_key_.ToHereTile();
-    ASSERT_FALSE(tile_result->IsSuccessful());
-    ASSERT_EQ(olp::client::ErrorCode::NotFound,
-              tile_result->GetError().GetErrorCode());
-    ASSERT_TRUE(tile_result->tile_key_.IsValid());
-  }
+  ASSERT_FALSE(response.IsSuccessful());
+  ASSERT_EQ(response.GetError().GetErrorCode(), client::ErrorCode::NotFound);
```

Comment:
when we requesting list of tiles without level we should receive list with result not found for tiles without data.

Reply:
reverted the behavior